pkg/minikube/command/ssh_runner.go

```diff
@@ -189,6 +196,49 @@ func (s *SSHRunner) Copy(f assets.CopyableFile) error {
 	return g.Wait()
 }
 
+func (s *SSHRunner) fileExistsInVM(f assets.CopyableFile, dst string) bool {
```

Comment:
Do you mind naming this something that hints that the function checks if the same file exists? Perhaps `sameFileExists`?

The `InVM` isn't necessary here. We may be SSH'ing into a container.